flake.nix

```diff
@@ -256,7 +256,7 @@
         pkgName,
         extraBuildPhase ? "",
         extraInstallPhase ? "",
-        dontStrip ? false,
+        dontStrip ? true,
```

Comment:
For the onlookers: we need minimum debug symbols for trace dumps. This is useful for the production services and not just local or spot debugging.